TEAM.md

```diff
@@ -0,0 +1,56 @@
+# Team membership
+
+Current members are:
+
+- `@alexbit-codemod`
```

Comment:
please add @mohebifar to the list here. you might need to send him a new invite for the node org since we missed the initial deadline. sry about that. thanks team.

Reply:
omg 3rd time? 🤦 sorry about that

Reply:
🎉🎉 Thank you! Joined.